src/util.ts

```diff
@@ -296,6 +297,21 @@ export function getToolsEnvVars(): any {
 	return Object.assign({}, process.env, toolsEnvVars);
 }
 
+export function getTestEnvVars(): any {
```

Comment:
`goTest.ts` would be a better place for this function as that file has all things test related.